ts/src/phemex.ts

```diff
@@ -53,7 +53,6 @@ export default class phemex extends Exchange {
                 'fetchDeposits': true,
                 'fetchFundingHistory': true,
                 'fetchFundingRate': true,
-                'fetchFundingRateHistories': false,
```

Comment:
same, why removing it here?

Reply:
the unified 'fetchFundingRateHistory' is the next line.  A code search for 'fetchFundingRateHistories' shows that it only is in 6 phemex files, all are derivative from this one.  Also, PR #13120 removed this from novadax, therock, and upbit over a year ago.